packages/core/content-manager/admin/src/pages/EditView/EditViewPage.tsx

```diff
@@ -121,6 +121,18 @@ const EditViewPage = () => {
     return transformDocument(schema, components)(form);
   }, [document, isCreatingDocument, isSingleType, schema, components]);
 
+  const validate = React.useCallback(
+    (values: Record<string, unknown>, options: Record<string, string>) => {
+      const yupSchema = createYupSchema(schema?.attributes, components, {
+        status,
+        ...options,
+      });
+
+      return yupSchema.validate(values, { abortEarly: false });
+    },
+    [components, schema?.attributes, status]
+  );
```

Comment:
Doesn't useDoc already create and return the `validate` function?

Reply:
True we could investigate if they can be shared. We need validation at the form level but we could try to plug the useDocument 🤔 

I think I'll remove this change from this PR and look into it separately